onvm/onvm_nflib/onvm_flow_table.h

```diff
@@ -71,8 +71,8 @@ struct onvm_ft {
 struct onvm_ft_ipv4_5tuple {
         uint32_t src_addr;
         uint32_t dst_addr;
-        uint16_t src_port;
-        uint16_t dst_port;
+        uint32_t src_port;
+        uint32_t dst_port;
```

Comment:
When running pktgen it default to a 5000+ size port number. This resulted in a seg fault. Im not to sure why this  was the case as it would only occur if adding a flow to the the shared flow table and not for example in the flow tracker which initializes a new flow table in the secondary process. However, increasing the size of the tuple ameliorated this occurence.

Reply:
what do you mean by `5000+ size port number`?  Increasing the size of a uint generally shouldn't fix a seg fault, but it suggests somewhere we might be allocating or memcpying into here something that is the wrong size.  In our other `key` datastructures do we use a 16 or 32bit port?  The IP specification says it should be 16 bit so it would be better to keep the struct that way.

Reply:
When creating a key with a port number greater that 1000+ a seg fault would occur. I printed the key that is created after a packet is arrived using onvm_ft_print_key, setting pktgen to have destination port of 1. However, I then noticed it would print the integer '256' after creating the key. In my last commit I converted the value to big endian. The correct port values are now being set and the seg fault no longer occurs.